src/countdown-timer/countdown-timer.tsx

```diff
@@ -96,6 +101,32 @@ export const CountdownTimer = ({
         }
     }, [wrapperRef.current]);
 
+    // Initial announcement effect
+    useEffect(() => {
+        if (isPlaying && remainingSeconds === timer) {
+            const initialMessage = `Starting countdown timer: ${getAccessibleTimeText(
+                remainingSeconds
+            )}`;
+            setInitialAnnouncement(initialMessage);
+        }
+    }, [isPlaying, remainingSeconds, timer]);
+
+    // Periodic announcements
+    useEffect(() => {
+        const timeElapsed = (timer || 0) - remainingSeconds;
```

Comment:
the consumer can set the countdown with either `timer` or `timestamp`, so the new calculation logic will also need to cater for that instead of checking `timer` directly

suggestion: enhance the `useTimer` hook to derive the initial seconds